### PR TITLE
*: replace dead more information links

### PR DESCRIPTION
**Pull request by gutjuri**
I ran the script from #5116 and fixed some of the outdated links. 
There are still some left, which I (or some other interested person) can fix in the future. I'll post an updated list of dead links in #5116 once this PR has been merged.

### CODE REVIEW

**Comment by gutjuri** [comment]
We could also add the script in #5116 to our Scripts/ directory. However, we might want to ask @sbrl first.

File: pages/linux/konsole.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # konsole
 
 > KDE's terminal emulator.
-> More information: <https://docs.kde.org/trunk5/en/konsole/konsole/command-line-options.html>.
+> More information: <https://konsole.kde.org>.
```
Comment (kbdharun) [inline pages/linux/konsole.md:4]:
```suggestion
> More information: <https://docs.kde.org/stable5/en/konsole/konsole/command-line-options.html>.
```

File: pages.zh/linux/konsole.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # konsole
 
 > Konsole: KDE 终端模拟器。
-> 更多信息：<https://docs.kde.org/trunk5/en/konsole/konsole/command-line-options.html>.
+> 更多信息：<https://konsole.kde.org>.
```
Comment (kbdharun) [inline pages.zh/linux/konsole.md:4]:
```suggestion
> 更多信息：<https://docs.kde.org/stable5/en/konsole/konsole/command-line-options.html>.
```

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution. I have manually checked/visited all the links, I think for the Konsole link alone we can link to the updated command line page instead of the main page as it doesn't contain any documentation for it (but we need to update it for every KDE release whereas your link would probably be permanent, so feel free to resolve it if you don't think the extra examples are necessary).

**Comment by kbdharun** [comment]
> If we have added it to the scripts-directory, we could also add it to tldr-maintenance to run on every commit on main. I have already done that for set-more-info-link.py. I could add set-alias-page.py, but it gives too many false positives if I am correct.

I would highly suggest against running it in a CI/scheduled action as it would cause unnecessary requests to the sites of smaller projects and overload their limited bandwidth (as mentioned by @SethFalco in a previous discussion we had to have something like this).

Having it in the script directory to be executed manually is fine but we should add disclaimers to make sure it is only run when absolutely necessary.

**Comment by gutjuri** [comment]
> LGTM, Thanks for your contribution. I have manually checked/visited all the links, I think for the Konsole link alone we can link to the updated command line page instead of the main page as it doesn't contain any documentation for it (but we need to update it for every KDE release whereas your link would probably be permanent, so feel free to resolve it if you don't think the extra examples are necessary).

I'll take the one provided by you.

**Pull request merged**